doc/config.md

```diff
@@ -0,0 +1,42 @@
+## Config the volcano device plugin binary
+
+The volcano device plugin has a number of options that can be configured. These options can be configured as command line flags, environment variables, or via a config file when launching the device plugin. The following section explains these configurations.
+
+### As command line flags or envvars
+
+| Flag                     | Envvar                  | Default Value   |
+|--------------------------|-------------------------|-----------------|
+| `--gpu-strategy`         | `$GPU_STRATEGY`         | `"number"`      |
+| `--config-file`          | `$CONFIG_FILE`          | `""`            |
+
+when starting volcano-device-plugin.yml, users can specify these parameters by adding args to the container 'volcano-device-plugin'.
+For example: 
+ - args: ["--gpu-strategy=share"] will let device plugin using the shared gpu strategy
```

Comment:
@peiniliu After "--gpu-strategy=share" is set, does the gpu-number support at the same time?

Reply:
I ask this question because we might merge another pr about vgpu. We want keep the two pr as consistent as possible.

Reply:
When gpu strategy share is set, the gpu number will be patched to the node(same as previous). the gpu number is there for volcano scheduler to generate memory for each device. I did not change any logic from the current version regarding the gpu sharing part(I mean if the pr works fine with current device plugin, they only need to specify the flag into share with the merged version).

If gpu number is set, the gpu memory will not post to the node, instead, the gpu number will be registered in kubelet rather than patched.

Reply:
> by the way, I checked https://github.com/volcano-sh/volcano/pull/2139/files. I saw they put a vgpuserver on pkg/scheduler/scheduler.go and do run and register there. I guess this should be a part of the improvement in the device plugin? but anyway, this pr is to support the configuration of the device plugin. The rest parts remain the same.

Yes, it's another improvement. Let's disscuss that in another pr.

Reply:
> l agree with @peiniliu , first it is unlikely to maintain both gpu-number and gpu-memory resources on the device plugin, and generally we do not recommend users mix the two resources.

It's ok to me to support them seperately at this stage.